DiagnosticsExtension/Models/ConnectionStringValidator/EventHubsValidator.cs

```diff
@@ -19,6 +21,7 @@ public class EventHubsValidator : IConnectionStringValidator
 
         public ConnectionStringType Type => ConnectionStringType.EventHubs;
 
+        ConnectionStringValidationResult.ManagedIdentityType identityType;
```

Comment:
Explicitly declare the scope, public/protected/private.  Same for all the validator classes.  It does not look like this should be public from what I can see.  The way its use as a way to conditionally pass information across function calls is not elegant, always prefer stateless functions - i.e. include this in the return type or exception instead of setting a class member as a way to communicate a return value.

Reply:
fixed